s3/settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Configuration for the S3 Interface microservice settings.
+"""
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+def get_s3_info():
+    vcap_services = os.getenv("VCAP_SERVICES", "")
+    try:
+        key_id = json.loads(vcap_services)["s3"][0]["credentials"]["access_key_id"]
+        secret_key = json.loads(vcap_services)["s3"][0]["credentials"]["secret_access_key"]
+        region = json.loads(vcap_services)["s3"][0]["credentials"]["region"]
+        bucket = json.loads(vcap_services)["s3"][0]["credentials"]["bucket"]
+    except (json.JSONDecodeError, KeyError) as err:
+        log.warning("Unable to load info from VCAP_SERVICES")
+        log.debug("Error: %s", str(err))
+        key_id = ""
+        secret_key = ""
+        region = ""
+        bucket = ""
+    
+    return (key_id, secret_key, region, bucket)
```

Comment:
Not that tuples are inherently bad, but you should always try to use the language's type-system to your advantage. This data **is** structured and has meaning, but the tuple hides that and doesn't allow programmers to name elements, while a class would.

A 4-element tuple isn't as meaningful to other programmers as a S3Credential class that had named attributes `.key_id, .secret_key, .region, .bucket`.